keyboards/rura66/rules.mk

```diff
@@ -0,0 +1,27 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
```

Reply:
@2Moons-JP
Hello.
How is your progress?